client/core/core.go

```diff
@@ -946,14 +1077,13 @@ func (c *Core) Trade(pw string, form *TradeForm) (*Order, error) {
 	}
 
 	// Fund the order and prepare the coins.
-	coins, err := fromWallet.Fund(qty, fromAsset)
+	coins, err := fromWallet.Fund(qty, wallets.fromAsset)
```

Comment:
For a buy order, where from wallet is the quote asset, does `qty` need to be in quote asset coins instead of base asset coins? Seems like this might be the trouble I was having with a buy order in issue https://github.com/decred/dcrdex/issues/231